### PR TITLE
Avoid using single-line cpp comments

### PR DESCRIPTION
**Pull request by rbuj**

### CODE REVIEW

**Comment by raveit65** [comment]
You need to call reviewers after opening a PR.

File: mate-panel/panel-background.c [hunk 1]
```diff
@@ -619,8 +619,9 @@ load_background_file (PanelBackground *background)
 	if (!g_file_test (background->image, G_FILE_TEST_IS_REGULAR))
 		return;
 
-	//FIXME add a monitor on the file so that we reload the background
-	//when it changes
+	/* FIXME add a monitor on the file so that we reload the background
+	 * when it changes
+         */
```
Comment (raveit65) [inline mate-panel/panel-background.c:624]: Indent...

File: mate-panel/panel-menu-items.c [hunk 2]
```diff
@@ -568,8 +568,8 @@ drive_poll_for_media_cb (GObject      *source_object,
 		g_error_free (error);
 	}
 
-	//FIXME: should we mount the volume and activate the root of the new
-	//mount?
+	/*FIXME: should we mount the volume and activate the root of the new
+	 * mount? */
```
Comment (raveit65) [inline mate-panel/panel-menu-items.c:572]: Can you please move the closing element `*/` to the next line?
Same style as all others.

**Comment by raveit65** [comment]
@rbuj 
Rest looks good.

**Comment by rbuj** [comment]
@raveit65 done

**Pull request merged**